src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,57 @@
+package duke.task;
+
+/**
+ * Task class.
+ */
+public abstract class Task {
+    protected boolean completed;
```

Comment:
I think isCompleted is a better name for a boolean?

Reply:
You're right! I have amended it.